test-suite-utils/src/main/java/io/microstream/testutils/LocalStackUtils.java

```diff
@@ -0,0 +1,28 @@
+package io.microstream.testutils;
```

Comment:
should this be `io.micronaut.microstream.testutils`?

Reply:
Renamed here https://github.com/micronaut-projects/micronaut-microstream/pull/190/commits/af2dcb7a55d9a5d6819e8c8587ce3ceaa65318b0

Reply:
thanks